packages/backend/src/plugins/search.ts

```diff
@@ -68,14 +69,18 @@ export default async function createPlugin({
   // particular collator gathers entities from the software catalog.
   indexBuilder.addCollator({
     defaultRefreshIntervalSeconds: 600,
-    collator: DefaultCatalogCollator.fromConfig(config, { discovery }),
```

Comment:
Thought unrelated to this PR - Additions like these (and below) to individual backend plugin files (packages/backend/*) always leave me wondering if deconstructing 'PluginEnvironment' adds any value ever. Keeping it just an 'env' object in all the Backstage ejected files would mean newer changes would need less manual changes from the adopter in order to use those changes.

Reply:
Yep, we totally want some DI in place as a more permanent solution to this too though :>